### DISCUSSION TITLE
DYNAMITE meeting today ? (June 28th)

### DISCUSSION THREAD

**Discussion by boeckera** [dynamics-of-stellar-systems/dynamite#397] [Announcements]
Hello @dynamics-of-stellar-systems/dynamite-maintainers ,

sorry but are we going to have a DYNAMITE meeting today?
I will be able to join online, but if there is nothing to discuss/no updates we can also skip it.
Please let me know!

Cheers,
Alina

**Comment by prashjet** [comment]
I have no urgent need to meet today, but happy to do so if anyone would like to discuss anything

**Reply by afeldmei** [reply]
Same, I am available but fine with skipping it. If we skip this week, will we meet next week or in 2 weeks? I will be at a conference in 2 weeks.

**Comment by boeckera** [comment]
I think the whole July is kind of a mess. I will only be there in the second week.

**Comment by prashjet** [comment]
I'm only around the last Friday in July

**Comment by maindlt** [comment]
I am fine either way.
Status coloring: reading pop data is done, but not the new binning via Fortran (in the works...)
Status pull requests: #383 (new installation procedure) is waiting for testing, a new one was necessary due to an update of plotbin #396 (Replace deprecated code) is ready for testing, too. We can handle both via Email/git...
